src/html/sandbox.html

```diff
@@ -78,14 +78,59 @@
 
             window.addEventListener('message', function (event) {
                 var command = event.data.command;
-                if (command === "render") {
+                if (command === "renderTemplates") {
                     event.source.postMessage({
+                        content: 'template-editor',
                         html: htmlTemplate(event.data.context),
                         callback: event.data.callback,
                         openTemplate: event.data.openTemplate
                     }, event.origin);
                 }
             });
         </script>
+
+        <script id="add_Domains" type="text/x-handlebars-template">
+            {{#each domains}}
+                {{#if this.default}}
+                    <li class='collection-item container custom-domain-collection'>
+                        <div class='row valign-wrapper remove-margin'>
+                            <div class='col s10 left-align custom-domain-label'>
+                                {{ this.name }} 
+                            </div>
+                            <div class='col s2'>
+                            </div>
+                        </div>
+                    </li>
+                {{else}}
+                    <li class='collection-item custom-domain-collection container'>
+                        <div class='row valign-wrapper remove-margin' style: >
+                            <div class="col s10 left-align custom-domain-label">
+                                {{ this.name }} 
+                            </div>
+                            <div class='col s2'>
+                                <a id='customDomainRemoveButton' class='btn-floating btn-jti tooltipped' data-position='left' data-delay='50' data-tooltip='Remove Domain'> 
```

Comment:
As mentioned elsewhere. If we embed the 'id' of the domain into a data attr (`data-id`) here, then we won't need to loop over all the domains when deleting one.

Reply:
:+1: